### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server-Certificates_Labels or Instructions: There is no label for the search box present beside the refresh button.

### ISSUE THREAD

**Issue by Shayanti123** [Microsoft/IIS.WebManager#69]
**Environment Details:**
URL: https://manage.iis.net/webserver/certificates
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**MAS ID Violated:** MAS3.3.2 

**Repro Steps:**
1) Download and install "Microsoft IIS Administration" application.
2) Enable Internet information Services.
3) Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4) Tab till Web Server and press enter
5) Tab till Certificates and press enter
6) Verify that the labels are present for input controls and also proper instructions are present to    guide the user as in how to input values in proper format.

**Actual Result:** 
There is no label for the search box present beside the refresh button and thus Screen Reader is   reading the search box as only editing, end of line

**Note:** Similar issue observed in Default Documents page

**Expected Result:** 
There should be label or some place holder text for the search box. User should be able to visually see labels for each control and also proper instructions for the same.

**User Impact:** 
User wont be able to understand the purpose of the form fields if screen reader does not read the label for the forms 

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachments:**
[Bug_69_There is no label present for the search box.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317521/Bug_69_There.is.no.label.present.for.the.search.box.pptx)

[Bug_69_Screen Reader is reading only editing, end of line for the search box.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317522/Bug_69_Screen.Reader.is.reading.only.editing.end.of.line.for.the.search.box.pptx)

[Bug_69_Proper label is not programmatically identified for the text box.pptx](https://github.com/Microsoft/IIS.WebManager/files/2326680/Bug_69_Proper.label.is.not.programmatically.identified.for.the.text.box.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #MAS3.3.2; #C+E-H; #.NETWebsites; #Chrome; #Edge

**Comment by fsteffi** [comment]
#A11ySev2;

**Comment by fsteffi** [comment]
#closed;
manage.iis.net is deprecated. The "Manage IIS" button on the [www.iis.net](http://www.iis.net/) home page will  be removed.
